src/main/resources/db/dev/data/V900_1__authorization_consent.sql

```diff
@@ -1,2 +1,5 @@
 INSERT INTO oauth2_authorization_consent(registered_client_id, principal_name, authorities) VALUES
  ('ceab6c18-081b-44e9-8130-3b37a254108f', 'test-client-id', 'ROLE_OAUTH_ADMIN,ROLE_AUDIT,ROLE_TESTING');
+
+INSERT INTO oauth2_authorization_consent(registered_client_id, principal_name, authorities) VALUES
+    ('83c3869b-696d-4a6e-8da4-3d36666bab9d', 'test-client-create-id', 'ROLE_VIEW_GROUPS');
```

Comment:
`test-client-create-id` was created as part of testing the OIDC registration end point. This client has to be used when accessing that end point. Probably better to use a different client for this